extensions/wikia/AdEngine/js/lookup/prebid/adapters/indexExchange.js

```diff
@@ -125,9 +115,8 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.indexExchange',[
 			},
 			bids: config.sizes.map(function (size) {
 				return {
-					bidder: bidderName,
+					bidder: requestsBidderName,
```

Comment:
Does it mean we need to retarget all line items?

Reply:
No, Index Exchange only requires `ix` in bid params because... just because. But we can rename all the lines, modules and test variables if we have to much free time :>